sql/comments_database_setup.sql

```diff
@@ -289,3 +289,24 @@ CREATE TABLE semesterInfo (
   updatedAt TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
   PRIMARY KEY (universityId, instanceId)
 );
+
+CREATE TABLE announcement(
+    id INT PRIMARY KEY AUTO_INCREMENT,
+    courseId VARCHAR(255) NOT NULL,
+    instructorId VARCHAR(255) NOT NULL,
+    instanceId VARCHAR(255) NOT NULL,
+    title VARCHAR(255) NOT NULL,
+    content TEXT NOT NULL,
+    visibleUntil TIMESTAMP NULL,
+    createdAt TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updatedAt TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+);
+
+CREATE TABLE courseMetadata (
+    courseId VARCHAR(50) NOT NULL,
+    instanceId VARCHAR(50) NOT NULL,
+    lectureSchedule JSON NOT NULL,
+    updaterId VARCHAR(255),
```

Comment:
hasHomework and hasQuiz is courseSpecific - it is true if the course has any quizzes or homework. See `CourseInfo` at `courseInfo.ts` 

A lecture entry can also have 'hasQuiz' - which indicates if there will be a quiz in during the lecture. 'hasHomework' doesn't currently make sense for a lecture entry